test/deno/stage2.test.ts

```diff
@@ -0,0 +1,57 @@
+import { assertEquals, assertStringIncludes } from 'https://deno.land/std@0.127.0/testing/asserts.ts'
+
+import { join } from 'https://deno.land/std@0.155.0/path/mod.ts'
+import { pathToFileURL } from 'https://deno.land/std@0.155.0/node/url.ts'
+
+import { getStage2Entry } from '../../deno/lib/stage2.ts'
+import { virtualRoot } from '../../deno/lib/consts.ts'
+
+Deno.test('`getStage2Entry` returns a valid stage 2 file', async () => {
```

Comment:
This test was long overdue because we're dynamically generating code so it's good that we validate it as early as possible.